components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -42,15 +42,20 @@ export default class InvitationModalMembersStep extends React.Component {
 
     copyLink = () => {
         const input = this.inviteLinkRef.current;
-        input.focus();
-        input.setSelectionRange(0, input.value.length);
+
+        const textField = document.createElement('textarea');
+        textField.innerText = input.value;
+
+        document.body.appendChild(textField);
```

Comment:
Here the textField is directly added to the body of the document. This might create some jiff effects as the element might be created and then removed for a moment.

Reply:
@srkgupta I agree, if we use `textField.style.display = 'none'` I think that will take care of the issue.

Reply:
actually `display: none` doesn't allow me to copy the content, so I used
```js
textField.style.position = 'fixed';
textField.style.opacity = 0;
```